pkg/console/util.go

```diff
@@ -456,6 +456,14 @@ func doInstall(g *gocui.Gui, hvstConfig *config.HarvesterConfig, webhooks Render
 		return err
 	}
 
+	// specific the node label for the specific node role
+	if hvstConfig.Role == config.RoleEtcd {
+		if hvstConfig.Labels == nil {
+			hvstConfig.Labels = make(map[string]string)
+		}
+		hvstConfig.Labels["node-role.harvesterhci.io/etcd"] = "true"
```

Comment:
`node-role.harvesterhci.io/etcd` is used around the code, it would be nice to make it a constant.